ui/lib/components/index.ts

```diff
@@ -44,8 +44,11 @@ export * from './InstanceSelect'
 export { default as InstanceSelect } from './InstanceSelect'
 export * from './MultiSelect'
 export { default as MultiSelect } from './MultiSelect'
-export * from './ShortValueWithTooltip'
-export { default as ShortValueWithTooltip } from './ShortValueWithTooltip'
+export * from './ValueWithTooltip'
+export {
+  default as ShortValueWithTooltip,
+  ScaledBytesWithTooltip,
```

Comment:
I would prefer to use `<ValueWithTooltip.Short />`, `<ValueWithTooltip.Bytes />`, etc.

Actually it can also be `<FormatValue model="plain|short|bytes|..." value="..." />`